app/views/ListRadiosView.scala.html

```diff
@@ -46,7 +46,7 @@ <h1 class="govuk-heading-l">@renderMessage(viewmodel.heading)</h1>
                   field = form("value"),
                   items = rows,
                   legend = viewmodel.page.legend
-                    .map(legend => Some(LegendViewModel(renderMessage(legend)).withSize(LegendSize.Small)))
+                    .map(legend => Some(LegendViewModel(renderMessage(legend)).withSize(LegendSize.Medium)))
```

Comment:
This is going to affect all pages using this view, including a choice page for removal (`WhichEmployerContributionRemoveController`), just wanted to double check if this is ok.

Reply:
I'll pull this out into a param

Reply:
Only if it's needed @westwater . If all usages need to be Medium, please keep it